dataline-singer/src/main/resources/json/SingerMessage.json

```diff
@@ -21,13 +21,14 @@
       "type": "string"
     },
     "record": {
-      "description": "record message: must me a JSON object",
+      "description": "record message: must be a JSON object",
       "type": "object",
       "existingJavaType": "com.fasterxml.jackson.databind.JsonNode"
     },
     "schema": {
       "description": "schema message: the schema",
-      "$ref": "SingerCatalog.json#/definitions/SingerColumnMap"
+      "type": "object",
```

Comment:
i thought the schema is a well defined object in the singer spec that matches the schema of the schema in the catalog jsonnot just a json blob. am i wrong about this?